### ISSUE TITLE
Keyboard shortcut not pressing character 1 skill even with CastAbility1ShortcutKeyCode=81

### ISSUE THREAD

**Issue by ramblinguy** [Masuzu/ZooeyBot#1235]
![image](https://user-images.githubusercontent.com/23442109/62426786-96654380-b6b7-11e9-8303-49b33160fd9e.png)

Sorry about the slightly clipped image. So I set my ini file to have these options:
CastAbility1ShortcutKeyCode=81
CastAbility2ShortcutKeyCode=87
Summon1ShortcutKeyCode=81
Summon2ShortcutKeyCode=87

and you can see that after pressing key 49 to select the first character, key 81 is never pressed. However, the first summon works correctly. 81 is pressed after 53 and the summon is cast. I have a QWERTY keyboard. Not sure if this is happening to anyone else, but it makes it unuseable for mechanic setups. Ability 2-4 is working fine, as is Ability 1 for all other characters. I am using the newest version of Zooey.

**Comment by Rkdheishins** [comment]
when u select use shortcut keys to true this doesn't work for me either, my workaround is putting that section as false and make the mouse move manually itll take a second to do but its just as good

**Comment by Masuzu** [comment]
Acknowledged
https://discordapp.com/invite/JaNR87n

**Comment by readmynameplease** [comment]
yeah got the same problem but the issue for me is it's not working on the MC only hope @Masuzu can address this.
